.gitignore

```diff
@@ -10,9 +10,12 @@
 *.json
 
 # conmech input/output
+log/*
 datasets/*
 output/*
 
+train.sh
```

Comment:
We should avoid adding specific files to gitignore. You can just don't add these files to git.